okio/jvm/src/test/java/okio/ThrottlerTest.kt

```diff
@@ -122,18 +52,6 @@ class ThrottlerTest {
     stopwatch.assertElapsed(0.25)
   }
 
-  @Test fun sourceAfterClear() {
```

Comment:
these `afterClear()` tests are less valuable now that we test synchronization/source/sink separately from the byte math.